examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -105,6 +101,16 @@ angular.module('cloudberry.common', [])
           values: keywords
         }
       ];
+      if (parameters.geoIds.length <= 2000){
```

Comment:
The length is the real number of geoIds that will be placed in the query. If you mean the number 2000, it is just a safe estimation._I tried it on the textdb server and it won't cause parse exceptions. I don't have an exact limit because the size limit seems to depend on the machine. The same query may work on one machine but fail on another. Even when I send the same query to the same machine, if the size is in a certain range, it may both work or fail.